ansible/roles/git_lfs/tasks/main.yaml

```diff
@@ -1,17 +1,3 @@
-# https://github.com/git-lfs/git-lfs/wiki/Installation#debian-and-ubuntu
-- name: Download script
-  ansible.builtin.get_url:
-    url: https://packagecloud.io/install/repositories/github/git-lfs/script.deb.sh
-    dest: /tmp/script.sh
-    mode: 0764
-
-- name: Setup
-  become: true
-  environment:
-    os: "{{ ansible_distribution }}"
-    dist: "{{ ansible_distribution_release }}"
-  ansible.builtin.script: /tmp/script.sh
-
 - name: Install Git LFS
```

Comment:
@yukke42 Should we add a new step for `git lfs install`?

Reply:
No, It's ok if CI passes.

Reply:
Right now in the ansible does:
`sudo apt install git-lfs` only.

Ansible doesn't do `git lfs install`.

So we should remove `git lfs install` from the readme.

Reply:
@yukke42 Is that okay? Do you have any alternative ideas?

Reply:
I will create a new PR to fix installation steps.

Reply:
please move to <https://github.com/autowarefoundation/autoware/pull/3029>.